### PR TITLE
M-13 Configure logger to send logs to logstash

### PR DESCRIPTION
**Pull request by mateuszjanczak**

### CODE REVIEW

**Review by szymoneg — APPROVED** [review]

**Review by kaburen — APPROVED** [review]

File: src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    <include resource="org/springframework/boot/logging/logback/console-appender.xml" />
```
Comment (Boryszs) [inline src/main/resources/logback-spring.xml:4]: Delete blank sign

File: src/main/resources/logback-spring.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                <logLevelValue/>
+                <stackTrace/>
+                <logstashMarkers/>
+            </providers>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="CONSOLE"/>
+        <appender-ref ref="LOGSTASH" />
```
Comment (Boryszs) [inline src/main/resources/logback-spring.xml:31]: Same as above

**Review by Boryszs — APPROVED** [review]

File: src/main/resources/logback-spring.xml (hunk 1 above)
Comment (mateuszjanczak) [reply src/main/resources/logback-spring.xml]: deleted

File: src/main/resources/logback-spring.xml (hunk 2 above)
Comment (mateuszjanczak) [reply src/main/resources/logback-spring.xml]: done

**Pull request merged**